linux/gen_and_run_parallel.sh

```diff
@@ -3,9 +3,14 @@ num_of_runs=$1
 circle_consts=$2
 matrix_consts=$3
 simulation_consts=$4
-output_folder=$5
+TIMESTAMP=`date --utc -d "+3 hours" +_%d%m%Y_%H`
+output_folder=$5$TIMESTAMP
```

Comment:
Why not use minutes and seconds also?

Reply:
i don't think we really need the minutes and seconds, up until now we only added the hour

Reply:
get what you say, but in the future it might cause a problem, so you should check if the dir exists or try to make it more unique. Maybe seconds is exgeration, but I think minutes is mandatory

Reply:
done